src/components/backend-ai-app-launcher.ts

```diff
@@ -1302,9 +1311,9 @@ export default class BackendAiAppLauncher extends BackendAIPage {
           <div style="padding:15px 0;">${_t('session.SFTPDescription')}</div>
           <section class="vertical layout wrap start start-justified">
             <h4>${_t('session.ConnectionInformation')}</h4>
-            <div><span>SSH URL:</span> <a href="ssh://127.0.0.1:${this.sshPort}">ssh://127.0.0.1:${this.sshPort}</a>
+            <div><span>SSH URL:</span> <a href="ssh://${this.sshHost}:${this.sshPort}">ssh://${this.sshHost}:${this.sshPort}</a>
```

Comment:
Current appproxy v1-based method should exist and new approach have to be another one.

Reply:
it is handled in https://github.com/lablup/backend.ai-webui/pull/1694/commits/eb4ddce3945a87471a5487874bc40b1edb0064af#diff-8bfdd81f6977f0f8e05c0ba087d666df8c36729464b9b50adad2acba20077326R911. I appreciated for your feedback.